Uplift/Views/GymDetail/Facilities/CourtView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  CourtView.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 11/3/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class CourtView: UICollectionViewCell {
+
+    // MARK: Gym
+    private var facilityDetail: FacilityDetail!
+    private var dailyGymHours: [DailyGymHours]!
+    private var selectedDayIndex = 0
+
+    // MARK: CollectionView
+    private let courtsCollectionView: UICollectionView
+    private var displayedHours: [FacilityHoursRange] = []
+    private let collectionViewIdentifier = "court"
+
+    // MARK: Display
+    private static let courtSize = CGSize(width: 124, height: 192)
+
+    override init(frame: CGRect) {
+        let flowLayout = UICollectionViewFlowLayout()
+        let spacing: CGFloat = -1 // Overlap Center Court Line
+        flowLayout.scrollDirection = .horizontal
+        flowLayout.minimumInteritemSpacing = 10000
+        flowLayout.minimumLineSpacing = spacing
+        flowLayout.itemSize = CourtView.courtSize
+
+        courtsCollectionView = UICollectionView(frame: .zero, collectionViewLayout: flowLayout)
+        courtsCollectionView.backgroundColor = .primaryWhite
+        courtsCollectionView.isUserInteractionEnabled = false
+        courtsCollectionView.allowsSelection = false
+        courtsCollectionView.isScrollEnabled = false
+        courtsCollectionView.register(CourtCell.self, forCellWithReuseIdentifier: collectionViewIdentifier)
+
+        super.init(frame: frame)
+        backgroundColor = .primaryWhite
+        courtsCollectionView.dataSource = self
+        addSubview(courtsCollectionView)
+        setupConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupConstraints() {
+        courtsCollectionView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+    }
+
+    override func layoutSubviews() {
+        centerCollectionView()
+    }
+
+    private func update(day: Int) {
+        selectedDayIndex = day
```

Comment:
you pass in `selectedDayIndex` for `day` and then just set `selectedDayIndex = day` which means you're just setting `selectedDayIndex = selectedDayIndex` right?